src/Pages/ArticleByTopic.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import ArticleCard from "../Components/ArticleCard";
+import { getArticles } from "../utils/api";
+
+export default function ArticleByTopic() {
+  const [articles, setArticles] = useState([]);
+  const [isloading, setIsLoading] = useState(true);
+
+  const { topic } = useParams();
+
+  useEffect(() => {
+    getArticles(topic).then(({ articles }) => {
```

Comment:
Same isLoading comment here, always good practice to 'reset' the isLoading state at the top of the useEffect as whenever it's running data is being fetched